src/ts/components/subtitleoverlay.ts

```diff
@@ -470,6 +477,24 @@ class ActiveSubtitleManager {
     }
   }
 
+  /**
+   * Removes all active cues which don't enclose the given time
+   * @param time the time for which subtitles should remain
+   */
+  public clearInactiveCues(time: number): ActiveSubtitleCue[] {
+    const removedEvents: ActiveSubtitleCue[] = [];
```

Comment:
NIT: Ar these cues or events? 😅

Reply:
It's actually a label and an event:
```ts
interface ActiveSubtitleCue {
  event: SubtitleCueEvent;
  label: SubtitleLabel;
}
```

Reply:
renamed in https://github.com/bitmovin/bitmovin-player-ui/pull/573/commits/10c0774097ff2b736a84630fdbe233fdba8bc81c